code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -44,6 +44,12 @@
 	var/delivery_icon = "deliverycloset" //which icon to use when packagewrapped. null to be unwrappable.
 	var/anchorable = TRUE
 	var/icon_welded = "welded"
+	/// Protection against weather (in this case, rad storms) that being inside of it provides.
+	var/weather_protection = list()
```

Comment:
Consider using a LAZYLIST instead of an empty list

Reply:
Make the values for this var bitflags and you don't need a list. If you really must use a list though allow null to be an acceptable value so that all subtypes don't need to be holding on to an empty list.

Reply:
I didn't use bitflags for this because this way it's consistent between mob weather resists and closet weather resists, so I wouldn't need to do a second set of defines. I'll make it so that null is an acceptable value.

Reply:
They can both be flags

Reply:
They cannot both be flags, as mobs can gain an immunity from multiple sources; bits on a bitflag cannot contain how many times a mob is immune to a weather, as the immunity is a single bit rather than an int. I looked into doing this as bitflags, but then came across this very problem I'm describing here.